sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -14,6 +14,20 @@ include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffs
 .Additional resources
 * xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user-provisioned cluster on bare metal]
 
+[id="checking-nodes-sandboxed-containers"]
+== Checking if you can deploy {sandboxed-containers-first} in your cluster
+
+Before running {sandboxed-containers-first} containers, you need to check whether the nodes in your cluster are eligible to run Kata containers.
```

Comment:
I'd consider adding a line or two of context here so that the thing is not so abstract for the user.  (OTOH most of the OpenShift docs are abstract in this sense so it might not be a problem.)  E.g.:

"Some worker nodes in a cluster might not comply with sandboxed containers' requirements for number of reasons, most prominently a lack of virtualisation support, and thus not be able to run sandboxed workloads.  To prevent installation of the sandboxed runtime on ineligible nodes and errors resulting from attempts to run sandboxed workloads on such nodes, you can install NFD to automatically check node eligibility."

Or something like that.

Reply:
Cool! Paraphrased this and added it.